### ISSUE TITLE
Is it possible to send a post request without headers?

### ISSUE THREAD

**Issue by JoshuaConcon** [vlucas/frisby#533]
Wondering if this is possible, since every single time I send a request with `headers: {}`, I get the following back (when I use `.inspectRequestHeaders()`.

```
Request Headers:
    Content-Type: application/json
    User-Agent: frisby/2.1.1 (+https://github.com/vlucas/frisby)
```

Is it possible to send an empty header?

**Comment by luuizeduardo** [comment]
If you are setting the headers with `.setup` function, you can pass a parameter to replace the default options.

Your request probably will be something like that:

![image](https://user-images.githubusercontent.com/36016665/63230494-d104cb00-c1e3-11e9-9be1-401fffdff126.png)

**Comment by vlucas** [comment]
Yes, the `frisby.setup()` method has a second argument to not merge, but set exactly what you provide.
